### ISSUE TITLE
Raspberry Pi build fails in timer

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#97]
i noticed this a couple of days ago, but wanted to make sure i was up to date.
I could offer a self-runner for pi if it would help. 

```
sudo apt upgrade
sudo apt update
cd maiko/bin
git pull
./makeright x

making so far for linux on armv7l.
making with display x releasename linux.armv7l-x ldename ldex
start making lde for linux.armv7l-x.
```
...

```
clang -m32 -std=gnu89 -funsigned-char -fno-strict-aliasing -fwrapv -c -O2 -g3 -DNEW_STORAGE -DLINUX -DAIX -DOLD_CURSOR -DBYTESWAP -DFORKCOMM -DNOFORN -DLOGINT  -DXWINDOW -DNOPIXRECT -DXV11R4 -DRELEASE=351 -DSYSVSIGNALS -DSYSVONLY  -I../inc/ -I../include/ ../src/timer.c  -o ../linux.armv7l-x/timer.o
../src/timer.c:531:3: warning: comparison between pointer and integer ('int' and '__sighandler_t' (aka 'void (*)(int)'))
  SIGERRCHK(sigset(SIGVTALRM, int_timer_service), "sigset vtalrm");
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/timer.c:117:11: note: expanded from macro 'SIGERRCHK'
  if (set == SIG_ERR) perror(str)
      ~~~ ^  ~~~~~~~
../src/timer.c:618:3: warning: comparison between pointer and integer ('int' and '__sighandler_t' (aka 'void (*)(int)'))
  SIGERRCHK(sigset(SIGIO, getsignaldata), "sigset io");
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/timer.c:117:11: note: expanded from macro 'SIGERRCHK'
  if (set == SIG_ERR) perror(str)
      ~~~ ^  ~~~~~~~
../src/timer.c:621:55: error: use of undeclared identifier 'F_SETSIG'
  if (fcntl(ConnectionNumber(currentdsp->display_id), F_SETSIG, 0) < 0)
                                                      ^
2 warnings and 1 error generated.
make: *** [makefile-tail:736: ../linux.armv7l-x/timer.o] Error 1
```

**Comment by waywardmonkeys** [comment]
I broke this. I will submit a PR within 18 hours or so.

**Comment by waywardmonkeys** [comment]
I didn't break it, but this is addressed by PR interlisp/maiko#104.

**Comment by nbriggs** [comment]
Having a self-runner for the Pi would be handy -- that and the Apple Silicon are the combinations of OS/CPU that we're targeting that I don't have any access to.  The Apple Silicon we won't be compiling native for, though, until XQuartz gets rebuilt.
